Permenant Protection Permission Issue (#5918)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/android-support-signin.md

Enable the required permission on Xiaomi devices.

- Display pop-up windows while running in the background.

> ## Unable to allow permission for 'Permanent protection' during onboarding, on some OEM devices

Reviewer: ```suggestion
## Unable to allow permission for 'Permanent protection' during onboarding on some OEM devices
```

> **Applies to:** Specific OEM devices only

Reviewer: ```suggestion
**Applies to:** Specific OEM devices only.
```

-   **Xiaomi with Android 11** 

> Defender App asks for Battery Optimization / Permanent Protection permission on the devices as part of app onboarding, and by clicking on "Allow" it brings back an error that the permission couldn't be set. It only affects the last permission called "Permanent Protection". 

Reviewer: ```suggestion
Defender App asks for Battery Optimization/Permanent Protection permission on devices as part of app onboarding, and selecting **Allow** returns an error that the permission couldn't be set. It only affects the last permission called "Permanent Protection." 
```

 
**Cause:**
Xiomi changed the battery optimization permissions in Android 11. Defender is not allowed to configure this setting to ignore battery optimizations.

**Solution:**

> We are working with OEMs for a solution to enable this permission form app onboarding screen. We will update the documentation as this is resolved.

Reviewer: ```suggestion
We are working with OEM to find a solution to enable this permission from the app onboarding screen. We will update the documentation when this is resolved.
```

> User can follow below steps to enable the same permissions form device settings.  

Reviewer: ```suggestion
Users can follow these steps to enable the same permissions from the device settings: 
```

1. Go to Settings on your device.

> 2. Search for "Battery Optimisation" and tap on "Battery Optimisation".

Reviewer 2: ![Search battery optimisation](https://user-images.githubusercontent.com/88181463/128720280-5b703a41-35a6-4358-b0a5-b6ed0da41e8e.png)

> 2. Search for "Battery Optimisation" and tap on "Battery Optimisation".

Reviewer 2: ![Search battery optimisation](https://user-images.githubusercontent.com/88181463/128720424-c97cbf17-b396-4c94-b4fc-0cf73889f408.png)

> 1. Go to Settings on your device.
> 2. Search for "Battery Optimization" and tap on "Battery Optimization".
> 3. On the Special app access, click on "Battery Optimization".
> 4. Change the Drop down to show all Apps.
> 5. Locate “Microsoft Defender Endpoint” and select “Don’t Optimize”.
> 
> Return to Defender onboarding screen, click on allow and you will be redirected to the dashboard screen.

Reviewer: ```suggestion
1. Go to **Settings** on your device.
2. Search for and select **Battery Optimization**.
3. In **Special app access**, select **Battery Optimization**.
4. Change the Dropdown to show **All Apps**.
5. Locate “Microsoft Defender Endpoint” and select **Don’t Optimize**.

Return to the Defender onboarding screen, select **Allow**, and you will be redirected to the dashboard screen.
```
